src/pages/post/[post_id].tsx

```diff
@@ -0,0 +1,232 @@
+
+                {/* date */}
+                <Stack spacing={2} direction="row" alignItems="center">
+                  <CalendarTodayIcon fontSize="large" />
+                  <Typography variant="body1">
+                    {post.startTime.format("DD/MM/YYYY h:mm A")} -{" "}
+                    {post.endTime.format("DD/MM/YYYY h:mm A")}
+                  </Typography>
+                </Stack>
+
+                {/* tags */}
+                <DisplayTags tags={post.tags.map((tag) => tag.name)} />
+
+                {/* description */}
+                <Stack spacing={0}>
+                  {post.description.split("\n").map((row, index) => (
+                    <Typography
+                      variant="body1"
+                      sx={{
+                        maxWidth: "30vw",
+                        wordBreak: "break-word",
+                        textAlign: "start",
+                      }}
+                      key={index}
+                    >
+                      {row}
+                    </Typography>
+                  ))}
+                </Stack>
+
+                {/* images */}
+                {post.description.split("\n").length < 6 && <DisplayImages images={post.images} />}
+              </Stack>
+            </CardContent>
+          </Card>
+          <Card sx={PostStyle.Card}>
+            <CardContent style={{ paddingTop: 0 }}>
+              <Stack spacing={2} alignItems="start" justifyContent="center">
+                {/* images */}
+                {post.description.split("\n").length > 5 && <DisplayImages images={post.images} />}
+
+                {/* participants */}
+                <Stack spacing={1} alignItems="start" justifyContent="center">
+                  <Typography variant="h2">Join with</Typography>
+                  <Box display="flex">
+                    {post.participants?.length == 0 && (
+                      <Typography variant="body1" color="error">
+                        No one is interested in this activity yet.
+                      </Typography>
+                    )}
+                  </Box>
+                  <Grid container spacing={1} style={{ marginLeft: -5 }}>
+                    {post.participants?.map((participant, index) => (
+                      <Grid item key={index}>
+                        <Participant participant={participant} />
+                      </Grid>
+                    ))}
+                  </Grid>
+                </Stack>
+              </Stack>
+            </CardContent>
+          </Card>
+        </Stack>
+
+        <Stack direction="row" spacing={4}>
+          <CommonButton
+            label="Delete Post"
+            color={COLOR.ERROR}
+            onClick={handleOpenDeletePostModal}
+          />
+          <CommonButton label="Edit Post" onClick={handleEditPost} />
+          <CommonButton
+            label="Create Appointment"
+            onClick={handleCreateAppointment}
+          />
+        </Stack>
+      </Stack>
```

Comment:
Can you add space below the button? Pic below is I scroll down to the end of page already.
<img width="1552" alt="image" src="https://user-images.githubusercontent.com/62459518/226830862-b6f5a361-565b-4d6f-bfdd-de3b03ba6382.png">